vba_timeseries/models/nonu/suuvid_obs_nonu.m

```diff
@@ -10,8 +10,9 @@
 phi = transform_phi(phi, inG);
 beta = phi(1); %motor speed recovery rate
 gamma = phi(2); %slope on vigor logistic (sensitivity)
-kappa = phi(3); %softmax temperature
-omega = phi(4); %stickiness
+nu = phi(3); %basal vigor
```

Comment:
You should be adding nu to value2pl/suuvid_obs_value2pl.m, not here. The 'nonu' folder should not have nu added to any aspect of it.